games/game_witcher3.py

```diff
@@ -11,20 +15,32 @@ class Witcher3Game(BasicGame):
 
     Name = "Witcher 3 Support Plugin"
     Author = "Holt59"
-    Version = "1.0.0a"
-    Description = "The Description Of The Dead"
+    Version = "1.0.1"
+    Description = "The Description Of The Dead| Updated to work with GOG by Luca/EzioTheDeadPoet"
 
     GameName = "The Witcher 3: Wild Hunt"
     GameShortName = "witcher3"
     GaneNexusHame = "witcher3"
```

Comment:
Wasn't this the initial reason for the PR?

Reply:
Yes, `GaneNexusHame` should be `GaneNexusName`, although it does not make any difference because the default for `GameNexusName` is `GameShortName`.